assets/js/script.js

```diff
@@ -38,12 +38,9 @@ function uploadFiles(){
         
         success: function(response){
             if(response.success){
-                var upload_prompt_div = $('#upload-prompt'),
-                    conversation_div = $('#whatsapp-conversation'),
-                    chat_div = conversation_div.find('#chat'),
-                    users_div = conversation_div.find('#users_list');
-
                 upload_prompt_div.hide();
+                back_nav.show();
+                logo_nav.addClass('nav-back');
```

Comment:
Why was adding and removing this class necessary to ```navbar-brand```

Reply:
This prevents a page reload if a user tries to go back by clicking the `.navbar-brand`. I suppose a better solution would be to permanently attach `.nav-back` to that element. Page reload is probably undesired behaviour for the user in any case.

Reply:
Yes. This is good :+1: